Stabikat.js

```diff
@@ -0,0 +1,255 @@
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function attr(docOrElem, selector, attr, index) {
+	var elem = index ? docOrElem.querySelectorAll(selector).item(index) : docOrElem.querySelector(selector);
+	return elem ? elem.getAttribute(attr) : null;
+}
+
+function typeMapper(type) {
+	switch (type) {
+		case 'Bücher' || 'Books':
+			return 'book';
+		case 'Briefe' || 'Letters':
+			return 'letter';
+		case 'Musikalien' || 'Music':
+			return 'music';
+		case 'Zeitschriften/Serien (ohne Online-Zeitschr.)' || 'Periodicals (non-online)':
+			return 'periodicals_non-online';
+		case 'Filme, Videos, etc.' || 'Audio visual':
+			return 'film';
+		case 'Tonträger' || 'Sound':
+			return 'audioRecording';
+		case 'Online-Zeitschriften' || 'Online periodicals':
+			// periodicals -- in contrast to their articles -- will use book-entries, as there is no specific type
+			return 'periodicals_online';
+		case 'Bilder' || 'Pictures':
+			return 'picture';
+		case 'Datenträger' || 'Software':
+			return 'Datenträger';
+		case 'E-Books/Online Ressourcen' || 'Online resources (without periodicals)':
+			// periodicals -- in contrast to their articles -- will use book-entries, as there is no specific type
+			return 'book';
+		case 'Kartenmaterial' || 'Cartography':
+			return 'map';
+		case 'Mikroformen' || 'Microfilm':
+			return 'microfilm';
+		case 'Aufsätze' || 'Articles':
+			return 'article';
+		case 'Manuskripte' || 'Handwriting':
+			return 'manuscript';
+		case 'Spiele, Skulpturen, etc.' || 'Games, Scupture, etc.':
+			return 'other';
+		default:
+			return 'book';
+	}
+}
+
+function detectWeb(doc, url) {
+	if (url.includes('ACT=SRCH') && getSearchResults(doc, true)) {
+		return 'multiple';
+	}
+	else {
+		var type = attr(doc, "#maticon", "alt");
+		return typeMapper(type);
+	}
+}
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
+			if (!items) {
+				return true;
+			}
+			var articles = [];
+			for (var i in items) {
+				articles.push(i);
+			}
+			ZU.processDocuments(articles, scrape);
+			return false;
+		});
+	}
+	else {
+		scrape(doc);
+	}
+}
+
+function scrape(doc) {
+	// get the identifaction number necessary for the api call
+	var permalink = attr(doc, ".cnt > table:nth-child(1) > tbody:nth-child(1) > tr:nth-child(4) > td:nth-child(1) > table:nth-child(1) > tbody:nth-child(1) > tr:nth-child(1) > td:nth-child(2) > a:nth-child(1)", "href");
```

Comment:
This is almost certainly going to break. (It probably already has.) See if there's an ID, class name, or attribute we could match on.